macros/GroupData.json

```diff
@@ -1834,10 +1857,15 @@
                             "RTCSessionDescriptionCallback",
                             "RTCStatsReport",
                             "RTCCertificate",
+                            "RTCIdentityProvider",
+                            "RTCIdentityAssertion",
```

Comment:
RTCIdentityProvider is an interface, which is indeed implemented. It's not documented yet although it's on the short list of things to get to soon.

You're right that RTCIdentityAssertion is a dictionary. I don't know yet if it's important enough to belong in the sidebar. I didn't catch that it's a dictionary or I'd have probably left it out for now. I'll remove it.